lagotto/etc/systemd/system/sidekiq.service

```diff
@@ -0,0 +1,19 @@
+{% from "lagotto/map.jinja" import props with context -%}
+{% set alm_root = props.get('app_root') -%}
+{% set rails_env = props.get('rails_env') -%}
+[Unit]
+Description=Lagotto sidekiq
+Requires=network.target
+
+[Service]
+Type=simple
+Environment="HOME=/home/lagotto"
+Environment="ALM_ROOT={{ alm_root }}"
+User=lagotto
+Group=lagotto
+ConditionPathIsDirectory={{ alm_root }}/current
+WorkingDirectory={{ alm_root }}/current
+ExecStart=/bin/bash -lc 'chruby-exec 2.2.2 -- bundle exec sidekiq -i 0 -e {{ rails_env }}'
```

Comment:
Same comment as above (-i 0) and ruby version. We pass the environment to Sidekiq, but you'll notice we don't for Lagotto which uses Puma (Puma's config (puma.rb) defines the Rails env).